README.md

```diff
@@ -54,12 +54,12 @@ Generate Release Notes action is dedicated to enhance the quality and organizati
 ### `row-format-issue`
 - **Description**: The format of the row for the issue in the release notes. The format can contain placeholders for the issue `number`, `title`, and issues `pull-requests`. The placeholders are case-sensitive.
 - **Required**: No
-- **Default**: `#{number} _{title}_ in {pull-requests}"`
+- **Default**: `"{number} _{title}_ in {pull-requests}"`
```

Comment:
This and also `row-format-pr` differs from the action.yml

Reply:
I do not see the difference.
The `#` has been moved to be part of {number} placeholder.

Reply:
Solved. You did not have a second `"` in a string of default value:`#{number} _{title}_ in {pull-requests}"`. That confused me.